### ISSUE TITLE
React-navigation-drawer cause a blank screen in creating build with eas

### ISSUE THREAD

**Issue by rafisamim** [react-navigation/react-navigation#11280]
### Current behavior

My app shows a white blank screen after the splash screen only in production mode, Its fine in expo go. After lots of debugging, I found out that it's because of the drawer and when I use the tab navigator the issue is fixed. The was fine in creating apk with expo build and the problem only found after using eas build. any idea? how to solve it?

**Comment by GAllwyn** [comment]
I am also facing the same issue. I am using expo sdk 47 with react-navigation v4. It is an expo app developed by somebody else, with rails backend deployed in Heroku. For 2 years, it was untouched. I upgraded it from sdk 39 to 47 incrementally, and migrating to EAS build. I have fixed all deprecation warnings and other errors. I followed the EAS build migration carefully. The app successfully build without any errors. When I open it, I am able to login and then the app crashes but the app works fine in expo go. I strongly feel it is due to drawer navigation. Any help would be appreciated. React-navigation documentation says support for react navigation v4 will still be available for latest compatible react native versions.
